sormas-e2e-tests/src/test/resources/features/sanity/web/Event.feature

```diff
@@ -453,4 +453,26 @@ Feature: Create events
     Then I check if downloaded data generated by custom event option is correct
     Then I delete exported file from Event Participant Directory
 
-
+  @issue=SORDEV-5967 @env_de
+  Scenario: Add evidence fields for event clusters
+    Given API: I create a new event
+    Then API: I check that POST call body is "OK"
+    And API: I check that POST call status code is 200
+    Given I log in with National User
+    And I click on the Events button from navbar
+    Then I open the last created event via api
+    And I check CLUSTER option on edit Event page
+    And I select Hauptsächlich von Mensch zu Mensch option from Primary Mode Of Transmission Combobox on edit Event page
+    And I click on Epidemiological evidence with UNBEKANNT option
+    And I click on Epidemiological evidence with NEIN option
+    And I click on Epidemiological evidence with JA option
+    And I tick the all options for Study on Epidemiological evidence for De version
+    And I tick the all options for Explorative survey of affected people on Epidemiological evidence for De version
+    And I tick the all options for Descriptive analysis of ascertained data on Epidemiological evidence for De version
+    And I tick the all options for Suspicion on Epidemiological evidence for De version
+    Then I click on Laboratory diagnostic evidence with UNBEKANNT option
+    And I click on Laboratory diagnostic evidence with NEIN option
+    And I click on Laboratory diagnostic evidence with JA option
+    And I tick the all options for Verification of at least two infected or diseased persons on Laboratory diagnostic evidence for De version
+    And I tick the all options for Verification on materials on Laboratory diagnostic evidence for De version
+    And I click on SAVE button in edit event form
```

Comment:
test is not completed, we select all options here, but we don't also check the test case checks after.

![image](https://user-images.githubusercontent.com/83643843/163781487-977d625d-f9ab-4b45-a613-b8f3d29a434f.png)